riscv.c

```diff
@@ -597,20 +644,20 @@ static bool op_system(struct riscv_t *rv, uint32_t inst)
             rv->io.on_ecall(rv);
             break;
         case 1:  // EBREAK
-            rv->io.on_ebreak(rv);
+            rv_breakpoint(rv, rv->PC);
```

Comment:
You should explain why the change against `EBREAK` is associated with RV32C.

Reply:
The use of `rv->io.on_ebreak(rv)` was a matter of object-oriented programming in C as it encapsulates the unnecessary implementation details.